templates/personal-settings.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/** @var $l \OCP\IL10N */
+/** @var $_ array */
+
+script('appointments', 'personal-settings');         // adds a JavaScript file
+style('appointments', 'settings');    // adds a CSS file
+?>
+
+<div id="appointments" class="section">
+        <h2><?php p($l->t('Appointments')); ?></h2>
+
+        
+    <div class="pub-page">
+        <p>
+            <?php p($l->t('Public Page URL')); ?>
+        </p>
+        <a href="<?php p($_['public-page-url']) ?>"  target="_blank">
```

Comment:
Could be possible to add a copy link button, I think it would be nice little quality-of-life improvement